docs/user-documentation/access-control.md

```diff
@@ -0,0 +1,144 @@
+# Access Control in Islandora
+
+_last updated on 2021-01-26_
+
+This page is about controlling **who can view or edit your Islandora content** (metadata and binaries). 
+
+Islandora recommends using Drupal's access control features. Contributed modules such as those described below, can provide additional flexibility and configurability. However, these only apply when content is accessed through Drupal, and are not applied to data in Fedora, the Triplestore, or Solr. 
+
+!!! danger "Exposed Endpoints"
+    If access control is a concern, you will want to lock down access to the following services that are part of the Islandora Suite:
+    
+    - Fedora (REST API and admin client)
+    - Solr (API and admin client)
+    - Blazegraph (API and admin client)
+    
+    as anyone accessing data through those services are bypassing Drupal's access control. In all out-of-the-box sandboxes and demo instances, these services are wide open.
+    
+## Access Control in Core Drupal
+
+**Summary: Core Drupal's access control features include the "published"/"unpublished" states and some basic permissions that can be granted globally or at the bundle level.**
+ 
+Drupal's concepts of "published" and "unpublished" for nodes and media apply to Islandora just as they do in Drupal. _Usually_, published content is visible to the entire world (without any authentication), while unpublished content can only be accessed by selected users - usually administrators, other privileged roles, and the user who created the content. This is configurable through the Drupal Permissions interface. 
+
+
+All Drupal permissions are granted to _roles_, not individual users. Read more [documentation on Users](/user-documentation/users). 
+
+The extent of the configurability of Drupal Core's access control (excluding revisions) is provided by the following permissions:
+
+| View permissions (apply to _all_ content or _all_ media regardless of bundle) |
+|---|
+| - View all published content |
+| - View own unpublished content |
+| - View (published) media |
+| - View own unpublished media |
+
+| "Editing" permissions (can be granted for a specific bundle, or for all nodes or media):  |
+|---  |
+| - Create new content |
+| - Edit (your) own content |
+| - Edit any(one's) content |
+| - Delete (your) own content |
+| - Delete any(one's) content |
+
+| Administrative permission (includes all the above and more, give to trusted users only): 
+| --- |
+| - Administer content  |
+
+Drupal Core does not include any mechanisms for the following:
+
+- individual nodes or media having their own access policies (in Drupal, access can only be configured at the content type or media type level)
```

Comment:
Could we have a line here about how to add this? Drupalorg has a fairly comprehensive page explaining and listing a bunch of access control contrib modules: https://www.drupal.org/node/270000

`Access control for individual nodes and media can be added via Drupal contributed modules. For more information, please see [Comparison and Overview of Access Control modules](https://www.drupal.org/node/270000) on drupal.org.`

Reply:
Scratch that, Serves me right for commenting before I read through the whole thing. I see there are contributed modules referenced later on. Maybe just a note and "see below" so users don't hit this line and take away that permission are site-wide, full stop, and don't keep going?

Reply:
Good point!